german/hoelzernes-Paddel/hoelzernes-Paddel.md

```diff
@@ -0,0 +1 @@
+Das hoelzerne Paddel erweist sich als sehr marode. Damit schaffen Sie es niemals bis ans Ufer. Nichtsdestotrotz sind Sie vorangekommen und sehen nun schon das Ufer. Sie hören Musik.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.